src/components/NFTFeatured/index.vue

```diff
@@ -121,10 +121,9 @@ export default {
   methods: {
     ...mapActions([
       'signLogin',
-      'addNFTFeatured',
-      'addNFTHidden',
-      'removeNFTFeatured',
-      'removeNFTHidden',
+      'setNFTFeatured',
+      'setNFTHidden',
+      'setNFTUnhidden',
```

Comment:
Maybe combine these actions into one, so you can call and handle the logic the same way in backend
```js
this.setNFTDisplayState({
  state: 'hidden', // 'featured', 'normal'
  address: this.getAddress,
  classId: this.classId,
})
```

Reply:
You may probably want to combine the mutations`NFT_SET_USER_NFT_CLASS_FEATURED_SET_MAP`
`NFT_SET_USER_NFT_CLASS_HIDDEN_SET_MAP` as well